src/components/AudioPlayer/AudioPlayerSlider.tsx

```diff
@@ -67,7 +70,7 @@ const AudioPlayerSlider = ({
           max={NUMBER_OF_STEPS}
         />
       </div>
-      <span className={styles.remainingTime}>{secondsFormatter(remainingTime)}</span>
+      <span className={styles.remainingTime}>{secondsFormatter(audioDuration)}</span>
```

Comment:
This causes the total time to be very bloated since `audioDuration` is in milliseconds and inside 
`secondsFormatter` we multiple it by 1000 again assuming that it's in seconds (since `currentTime` is in seconds). We have a couple of options here:

1. Keep using `audioDuration` but add a param inside `secondsFormatter` called `isInSeconds` (maybe also rename it to `timeFormatter` or something) to indicate whether the time that was passed to the function is in milliseconds or seconds format and when it's in milliseconds, the following line:

`let formattedTime = new Date(seconds * 1000).toISOString().substr(12, 7);`

should be replaced by:

`let formattedTime = new Date(isInSeconds? time * 1000: time ).toISOString().substr(12, 7);`

2. Use `audioDurationMilliSeconds` instead of `audioDuration` because `audioDurationMilliSeconds` is in seconds already. (Either way`audioDurationMilliSeconds` is a typo that I fixed in the smooth slider PR that wasn't merged and should be renamed to `audioDurationSeconds` because the value has been converted to seconds already.)

Option 2 is slightly worse because we have the value converted already from milliseconds to seconds then inside `secondsFormatter` we will convert it back to milliseconds.